### ISSUE TITLE
Tfsec says there is a critical vulnerability in variables.tf

### ISSUE THREAD

**Issue by mbellqq** [terraform-aws-modules/terraform-aws-rds#354]
## Description

The latest release of tfsec (0.58.4) identifies (perhaps incorrectly) this repo as having a couple of issues:
```
  Result 1

  [general-secrets-sensitive-in-variable][CRITICAL] Variable 'variable.password' includes a potentially sensitive default value.
  terraform-aws-rds/variables.tf:129


     126 | variable "password" {
     127 |   description = "Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file"
     128 |   type        = string
     129 |   default     = ""    string: ""
     130 | }
     131 | 
     132 | variable "port" {

  Legacy ID:  GEN001
  Impact:     Default values could be exposing sensitive data
  Resolution: Don't include sensitive data in variable defaults

  More Info:
  - https://tfsec.dev/docs/general/secrets/sensitive-in-variable#general/secrets 
  - https://www.terraform.io/docs/state/sensitive-data.html 

  Result 2

  [aws-rds-encrypt-instance-storage-data][HIGH] Resource 'module.db_instance:aws_db_instance.this[0]' has storage encrypted set to false
  terraform-aws-rds/modules/db_instance/main.tf:30


      27 |   instance_class    = var.instance_class
      28 |   allocated_storage = var.allocated_storage
      29 |   storage_type      = var.storage_type
      30 |   storage_encrypted = var.storage_encrypted    bool: false
      31 |   kms_key_id        = var.kms_key_id
      32 |   license_model     = var.license_model
      33 | 

  Legacy ID:  AWS052
  Impact:     Data can be read from RDS instances if compromised
  Resolution: Enable encryption for RDS instances

  More Info:
  - https://tfsec.dev/docs/aws/rds/encrypt-instance-storage-data#aws/rds 
  - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance 
  - https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html 

  Result 3

  [general-secrets-sensitive-in-local][CRITICAL] Local 'locals.' includes a potentially sensitive value which is defined within the project.
  terraform-aws-rds/main.tf:2


       1 | locals {
       2 |   master_password      = var.create_db_instance && var.create_random_password ? random_password.master_password[0].result : var.password    string: ""
       3 |   db_subnet_group_name = !var.cross_region_replica && var.replicate_source_db != null ? null : coalesce(var.db_subnet_group_name, module.db_subnet_group.db_subnet_group_id, var.identifier)
       4 | 
       5 |   parameter_group_name_id = var.create_db_parameter_group ? module.db_parameter_group.db_parameter_group_id : var.parameter_group_name

  Legacy ID:  GEN002
  Impact:     Local value could be leaking secrets
  Resolution: Don't include sensitive data in locals

  More Info:
  - https://tfsec.dev/docs/general/secrets/sensitive-in-local#general/secrets 
  - https://www.terraform.io/docs/state/sensitive-data.html 
```

Some of these may be false-positives in which case `#tfsec:ignore/XXXXX` comment tags could be added. But if not then can these be addressed?

## Versions
Terraform: 1.0.0
 - Provider(s):
provider.aws v3.37.0

## Reproduction

 - Clone the repo: `git clone https://github.com/terraform-aws-modules/terraform-aws-rds.git`
 - Download the latest tfsec, e.g. `wget https://github.com/liamg/tfsec/releases/latest/download/tfsec-linux-amd64 -O tfsec`
 - Run `./tfsec`

**Comment by nantiferov** [comment]
I would say first and third are valid and probably it's worth to mark them sensitive.
Second one is more like false positive.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.0.0 :tada:
